config.yaml

```diff
@@ -72,3 +72,8 @@ options:
       automatically deduced from it).
       See https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
     type: string
+  remote_write_receiver:
```

Comment:
I think we shouldn't have this config option. Instead, always enable remote write:
Currently, when we enable remote-write, it's enabled for everyone; afaict, there isn't greater risk in enabling it to begin with.